Assets/MRTK/Services/InputSimulation/InputSimulationService.cs

```diff
@@ -158,6 +158,11 @@ public void ResetHandRight()
         private bool wasFocused;
         private bool wasCursorLocked;
 
+        /// <summary>
+        /// Cached selectAction used to allow the user to select objects without needing the hands to be up when in gestures mode
+        /// </summary>
+        private MixedRealityInputAction selectAction;
```

Comment:
Is this something we'd want to put in the input simulation profile? That way it's configurable, instead of just searching the actions for "select"? Then could be grabbed out of the profile below like `SimulateEyePosition` is.